### ISSUE TITLE
An illegal reflective access operation has occurred

### ISSUE THREAD

**Issue by semuxdev** [semuxproject/semux#641]
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by io.netty.util.internal.ReflectionUtil to constructor java.nio.DirectByteBuffer(long,int)
WARNING: Please consider reporting this to the maintainers of io.netty.util.internal.ReflectionUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

This has been fixed by https://github.com/netty/netty/pull/7650.

Suggest upgrading our netty version when a new release comes out.

**Comment by semuxdev** [comment]
This has been resolved in netty-4.1.21; however, another illegal access was introduced

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by io.netty.resolver.dns.DnsNameResolver (file:/Users/yulongwu/.m2/repository/io/netty/netty-all/4.1.21.Final/netty-all-4.1.21.Final.jar) to method sun.net.dns.ResolverConfiguration.open()
WARNING: Please consider reporting this to the maintainers of io.netty.resolver.dns.DnsNameResolver
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by cryptokat** [comment]
Hey @semuxdev https://github.com/semuxproject/semux/pull/692 needs to be merged before we can close this issue.
